netlify/serve

```diff
@@ -1,10 +1,10 @@
-#!/bin/bash
+#bin/bash
```

Comment:
Why was the `!/` removed? `#!` is required for [shebangs](https://en.wikipedia.org/wiki/Shebang_(Unix))

Reply:
Oops..didn't mean to do that. Thanks for catching it! Fixed.